code/tfspkl_utils.py

```diff
@@ -193,8 +190,15 @@ def get_electrode_labels(conversation_dir):
     if not os.path.exists(header_file):
         return
```

Comment:
Should this be an error? What happens downstream if header file doesn't exist?

Reply:
Really redundant if you ask me. The header almost always exists. :)